app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt

```diff
@@ -260,6 +260,7 @@ class OnlineLibraryFragment : BaseFragment(), FragmentActivityExtensions {
         allowInternetPermissionButton.visibility = View.GONE
         librarySwipeRefresh.isRefreshing = false
       }
+      else -> {}
```

Comment:
Why there is else branch where `NetworkState` has only two states?

Reply:
In latest Kotlin we have to provide else otherwise its prompt error.